src/pages/SearchResult/SearchResult.scss

```
@@ -0,0 +1,76 @@
+* {
+  border: solid;
+}
+div.searchResultMain {
+  margin-top: 60px;
+  border: solid;
+  div.searchResultHead {
```

```suggestion
.searchResultMain {
  margin-top: 60px;
  border: solid;
  
  .searchResultHead {
```
- 불필요한 선택자는 제거하셔도 될 것 같습니다
- 새로운 선택자 앞에서는 한줄 띄워 구분해주세요

---

@glorious-min 리뷰 내용 적용해주세요!